src/mono/cmake/config.h.in

```diff
@@ -630,9 +585,6 @@
 /* Host is RISC-V */
 #cmakedefine HOST_RISCV 1
 
-/* Host is 32-bit RISC-V */
-#cmakedefine HOST_RISCV32 1
```

Comment:
I'd keep this one to have parity with TARGET_RISCV32. we always have both HOST_ and TARGET_ defines for architectures